src/demos/trackVehicle/ModelDefs.h

```diff
@@ -62,15 +62,23 @@ enum Enum {
 // relative to the chassis when facing forward.
 enum VehicleSide { RIGHTSIDE, LEFTSIDE };
 
+// specify what type of output 
+enum DebugType {
```

Comment:
This enum is treated as a namespace on lines 56 and 104 of ChTrackVehicle.cpp, but is not defined as such.